.gitignore

```diff
@@ -1,30 +1,5 @@
 /node_modules
-/LBRY-darwin-x64
 /dist
-
-/src/main/dist
-/src/main/locales
-/src/main/node_modules
-/src/renderer/dist
-/build/venv
 /build/daemon.ver
-/lbry-app-venv
-/lbry-app
-/lbry-venv
 /static/daemon/lbrynet*
-/static/locales
-/daemon/build
-/daemon/venv
-/daemon/requirements.txt
-/.idea
```

Comment:
this is here because many of us use JetBrains, which creates that dir

Reply:
I think it's better to keep project related ignore rules into the project `.gitignore`and user-specific rules into the user's global `.gitignore`.